requirements.txt

```diff
@@ -0,0 +1,2 @@
+requests==2.12.4
```

Comment:
Stripe specifies the minimum version (0.8.8) . . . 
Looking into this a little more and I'm not 100% sure, but it looks like when pip is reading a requirements.txt, it'll detect transitive dependencies, but pip freeze makes no distinction.